yaml_files/simphony_metadata.yml

```diff
@@ -266,6 +266,19 @@ CUDS_KEYS:
     CUBA.POWER_LAW_INDEX:
       default: 0.5
 
+  BINGHAM_PLASTIC_MODEL:
+    definition: Bingham plastic viscosity model (only in mixture model)
+    parent: CUBA.RHEOLOGY_MODEL
+    models: [CUBA.CONTINUUM]
+    CUBA.LINEAR_CONSTANT:
+      shape: (2)
```

Comment:
I would use [] instead. () is valid as well, but we use it only when we have a : entry